packages/flutter/lib/src/widgets/scroll_simulation.dart

```diff
@@ -120,8 +120,16 @@ class ClampingScrollSimulation extends Simulation {
     _distance = (velocity * _duration / _kInitialVelocityPenetration).abs();
   }
 
+  /// The position of the particle at the beginning of the simulation.
   final double position;
+
+  /// The velocity at which the particle is traveling at the beginning of the
+  /// simulation.
   final double velocity;
+
+  /// The amount of friction the particle experiences as it travels.
+  ///
+  /// The more friction the particle experiences, the faster it slows down.
```

Comment:
"the sooner it stops" is probably less odd-sounding that "the faster it slows down". :-)